plugins/modules/rds_cluster.py

```diff
@@ -389,6 +405,23 @@
     username: "{{ username }}"
     cluster_id: "cluster-{{ resource_prefix }}-restored"
     snapshot_identifier: "cluster-{{ resource_prefix }}-snapshot"
+
+- name: Create an Aurora PostgreSQL cluster and attach an intance
+  rds_cluster:
```

Comment:
```suggestion
 amazon.aws. rds_cluster:
```

Reply:
I would prefer to also to this later in another PR since we need to go through pretty much the whole test-suite anyway.

Reply:
Oh, it's the EXAMPLE block, not the test-suite. You're right, let's fix that.